docs/source/hi/_redirects.yml

```diff
@@ -0,0 +1,14 @@
+# Move "how-to" pages to the guides/ folder
+how-to-cache: guides/manage-cache
+how-to-discussions-and-pull-requests: guides/community
+how-to-downstream: guides/download
+how-to-inference: guides/inference
+how-to-manage: guides/repository
+how-to-model-cards: guides/model-cards
+how-to-upstream: guides/upload
+search-the-hub: guides/search
+guides/manage_spaces: guides/manage-spaces
+package_reference/inference_api: package_reference/inference_client
+
+# Alias for hf-transfer description
+hf_transfer: package_reference/environment_variables#hfhubenablehftransfer
```

Comment:
This is a file used in the English version of the docs to create redirects between old legacy URLs and new ones. We use it when we rename a file for example. So since Hindi documentation is brand new, no need to add this file as no legacy documentation exists.
TL;DR: this file can be removed :)